sycl/test-e2e/Graph/Inputs/add_nodes_after_finalize.cpp

```diff
@@ -85,8 +85,10 @@ int main() {
   free(PtrC, Queue);
   free(PtrOut, Queue);
 
-  assert(ReferenceC == DataC);
-  assert(ReferenceOut == DataOut);
+  for (unsigned i = 0; i < Size; i++) {
```

Comment:
Should we unify `unsigned` or `size_t` loop variables throughout the tests?

Reply:
unified on `size_t` since that's what the type of `Size` is